control/patch.yaml

```diff
@@ -0,0 +1,41 @@
+openapi: 3.1.0
+
+info:
+  title: Control patch models
+  description: >-
+    All control patch schemas
+  version: ^0.0.0
+
+components:
+  schemas:
+    Control.Patch:
+      description: >-
+        Request for triggering action on resources in traffic generator
+      type: object
+      properties:
+        choice:
+          type: string
+          enum: 
+          - flows
+        flows:
+          $ref: '#/components/schemas/Flows.Patch'
+    Flows.Patch:
+      description: >-
+        A container of flows with associated properties to be updated without
+        affecting the flows current transmit state.
+      type: object
+      required: [property_names, flows]
+      properties:
+        property_names:
+          description: >-
+            Flow properties to be updated without affecting the transmit state.
+          type: array
+          items:
+            type: string
+            enum: [rate, size]
+        flows:
+          description: >-
+            The list of configured flows for which given property will be updated.
+          type: array
+          items:
+            $ref: '../flow/flow.yaml#/components/schemas/Flow'
```

Comment:
The reason there's no dedicated data structure for updating flow properties like `size` and `rate` is because from SDK UX POV, it's easier for users to work with.

```python
c = api.config()
f1 = c.flows.add(name='f1')
f1.size.fixed = 256
f1.rate.pps = 500
f2 = c.flows.add(name='f2')
f2.size.fixed = 256
f2.rate.pps = 500

# set config for the first time
api.set_config(c)

# update rate for flow1
f1.rate.pps = 1000

cp = api.control_patch()
cp.flows.property_names = ["rate"]
cp.flows.flows = [f1]
api.set_control_patch(cp)

# fetch newly updated config
cpatched = api.get_config()

# fetched config shall be equivalent to what user already has
assert c == cpatched
```

Reply:
It looks like we should probably be explicit that only the values set in specified fields ( one of rate or size for flows ) will be patched with current config only for specified flows. The values of all remaining fields in the provided flows will be ignored ( or validated that no other fields are changes  ?) . I assume if no changes are done for corresponding attribute , then no separate warning is expected to be provided.